frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -561,6 +562,26 @@ export { useFlag } from '../utils/flags';
 export const ResourceYAMLEditor: React.FC<ResourceYAMLEditorProps> = require('@console/internal/components/AsyncResourceYAMLEditor')
   .AsyncResourceYAMLEditor;
 
+
+/**
+ * A basic YAML editor with hover help and completion.
+ * @example
+ * ```tsx
+ *   <YAMLEditor
+ *     value={code}
+ *   />
+ * ```
+ * @param {YAMLEditorProps['value']} - value - String representing the yaml code
```

Comment:
How does the editor response when `value` changes? Is this just the initial value? If so, should we call the prop `initialValue`?

Need a more complete description here.

Reply:
It's the [react-monaco-editor value prop](https://github.com/react-monaco-editor/react-monaco-editor#properties)